addons/website_mail_channel/controllers/main.py

```diff
@@ -3,10 +3,9 @@
 import datetime
```

Comment:
you may import openerp.fields' Date and Datetime classes and use from_string/to_string functions of theirs to perform date operations.